teku-testnet.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.6'
+
+####
+# https://github.com/ConsenSys/teku/tree/master/test-network
+
+services:
+  teku:
+    image: "${DOCKER_TEKU_IMAGE}"
```

Comment:
```suggestion
    image: "${DOCKER_TEKU_IMAGE:-consensys/teku:21.2}"
```

Reply:
I want to avoid this, as we use a centralized env file
we want to have a lower surface of refactoring when some update apear

Reply:
env file will overide this anyways